### PR TITLE
Generic object roles subcommand group

### PR DESCRIPTION
**Pull request by mdellweg**

### CODE REVIEW

File: pulpcore/cli/core/content_guard.py [hunk 1]
```diff
@@ -51,6 +52,7 @@ def rbac(ctx: click.Context, pulp_ctx: PulpContext) -> None:
 rbac.add_command(show_command(decorators=lookup_options))
 rbac.add_command(update_command(decorators=lookup_options))
 rbac.add_command(destroy_command(decorators=lookup_options))
+rbac.add_command(role_command(decorators=lookup_options))
```
[the remark this answers is not in the document]
Comment (mdellweg) [reply pulpcore/cli/core/content_guard.py:55]: That's a good observation.

**Comment by mdellweg** [comment]
@gerrod should be ready now.

**Comment by gerrod** [comment]
> @gerrod should be ready now.

Cool! But wrong guy, I don't know what a pulp is.

File: pulpcore/cli/common/generic.py [hunk 2]
```diff
@@ -79,12 +79,22 @@ class PulpGroup(click.Group):
     def __init__(
         self,
         *args: Any,
+        needs_plugins: Optional[List[PluginRequirement]] = None,
         allowed_with_contexts: Optional[Tuple[Type[PulpEntityContext]]] = None,
         **kwargs: Any,
     ):
+        self.needs_plugins = needs_plugins
```
Comment (gerrod3) [inline pulpcore/cli/common/generic.py:86]: :+1:

**Review by gerrod3 — APPROVED** [review]
Looks good!

**Pull request merged**